worker/worker.go

```diff
@@ -39,22 +39,23 @@ import (
 )
 
 type (
-	// Worker hosts workflow and activity implementations.
-	// Use worker.New(...) to create an instance.
+	// Worker is the entity that hosts and executes Workflow Tasks and Activity Tasks.
+	// Use worker.New(...) to create a new instance.
 	Worker interface {
 		Registry
 
-		// Start the worker in a non-blocking fashion.
+		// Start the Worker in a non-blocking fashion.
 		Start() error
 
-		// Run the worker in a blocking fashion. Stop the worker when interruptCh receives signal.
-		// Pass worker.InterruptCh() to stop the worker with SIGINT or SIGTERM.
-		// Pass nil to stop the worker with external Stop() call.
-		// Pass any other `<-chan interface{}` and Run will wait for signal from that channel.
-		// Returns error only if worker fails to start.
+		// Run the Worker in a blocking fashion.
+		// The Worker stops when interruptCh receives a signal.
+		// Pass worker.InterruptCh() to stop the Worker with SIGINT or SIGTERM.
+		// Pass nil to stop the Worker with an external Stop() call.
```

Comment:
> Pass nil to stop the Worker with an external Stop() call.

Can make people think they couldn't use `Stop()` _unless_ they pass nil. Really you don't have to "Pass nil to stop the Worker with an external Stop() call", you can stop the worker w/ such a call always.